arlo_server/ballot_manifest.py

```diff
@@ -0,0 +1,157 @@
+import csv, io, locale, uuid
+from sqlalchemy.orm.session import Session
+from flask import request, jsonify, Request
+from werkzeug.exceptions import BadRequest
+from datetime import datetime
+
+from arlo_server import app
+from arlo_server.models import (
+    db,
+    Batch,
+    Election,
+    File,
+    Jurisdiction,
+)
+from arlo_server.auth import with_jurisdiction_access
+from util.process_file import (
+    process_file,
+    serialize_file,
+    serialize_file_processing,
+    UserError,
+)
+
+BATCH_NAME = "Batch Name"
+NUMBER_OF_BALLOTS = "Number of Ballots"
+STORAGE_LOCATION = "Storage Location"
+TABULATOR = "Tabulator"
+
+
+def process_ballot_manifest_file(
```

Comment:
this method was basically copy-pasted from `util.ballot_manifest` - there's no new logic/changes to logic here